aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -1191,6 +1191,7 @@ Tensor & index_select_out_cpu_(const Tensor & self, int64_t dim, const Tensor &
   dim = maybe_wrap_dim(dim, self.dim());
   auto numel = index.numel();
   TORCH_CHECK_INDEX(index.dim() <= 1, "index_select(): Index is supposed to be a vector");
+  TORCH_CHECK(!(self.dim() == 0 && index.element_size() > 1), "index_select(): Index to scalar cannot have multiple values.");
```

Comment:
Hmm, it should be a `numel` rather than `element_size`, isn't it?

Reply:
Oh sorry, you are right. I thought `element_size` is an alias to `numel`.